From: David Steele <steele@debian.org>
Date: Thu, 29 Dec 2016 21:14:22 -0500
Subject: Add Comment to desktop file

To quiet an error reported by appstream.debian.org.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 08ee392..178cc7e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -247,8 +247,8 @@ install( FILES translations/fr_FR.qm DESTINATION ${CMAKE_INSTALL_DATADIR}/sirika
 file( WRITE ${PROJECT_BINARY_DIR}/sirikali.desktop
 
 "[Desktop Entry]
-Comment[en_US]=
-Comment=
+Comment[en_US]=Manage user encrypted volumes
+Comment=Manage user encrypted volumes
 Exec=${CMAKE_INSTALL_FULL_BINDIR}/sirikali -d %U
 GenericName[en_US]=Encrypted Folder Manager
 GenericName=Encrypted Folder Manager
